private/buf/cmd/buf/command/curl/curl.go

```diff
@@ -362,7 +373,40 @@ one is provided`,
 		userAgentFlagName,
 		"A",
 		"",
-		`The user agent string to send`,
+		`The user agent string to send. This is ignored if a -H or --header flag is provided
+that sets a header named 'User-Agent'.`,
+	)
+	flagSet.StringVarP(
+		&f.UserCreds,
+		userCredsFlagName,
+		"u",
+		"",
+		`The user credentials to send, via a basic authorization header. The value should be
+in the format "username:password". If the value has no colon, it is assumed to just be
+the username, in which case you will be prompted to enter a password. This overrides
+the use of a .netrc file. This is ignored if a -H or --header flag is provided that sets
```

Comment:
nit: if we're being pedantic, you should use `fmt.Sprintf` and pass in `headerFlagShortName` and `headerFlagName` (also there looks like there's no `headerFlagShortName` right now)

Reply:
done